src/ptc/util/misc.clj

```diff
@@ -125,3 +125,14 @@
     (when (nil? value)
       (throw (IllegalStateException. (str name " must not be nil"))))
     value))
+
+(defn parse-gs-url
+  "Return BUCKET and OBJECT from a gs://bucket/object URL."
+  [url]
+  (let [[gs-colon nada bucket object] (str/split url #"/" 4)]
+    (when-not
+      (and (every? seq [gs-colon bucket])
+           (= "gs:" gs-colon)
+           (= "" nada))
```

Comment:
```suggestion
  (let [[_ _ bucket object] (str/split url #"/" 4)]
    (when-not
      (and (seq bucket) (str/starts-with? url "gs://"))
```
I don't know if this is much cleaner, sorta just practicing my Clojure and thought I'd suggest it

Reply:
Looks like my suggestion got messed up, so nevermind 🙂